src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/SymbolTable.cs

```diff
@@ -1766,72 +1770,72 @@ private void SetParameterAttributes(MethodOrPropertySymbol methProp, ParameterIn
                     if (defType == typeof(byte))
```

Comment:
If I start doing that I'll end up about 3000 LOC changes in and deciding to pull back on the PR again. Which has been happening a lot ;)